google-cloud-datastore/src/main/java/com/google/cloud/datastore/IncompleteKey.java

```diff
@@ -77,34 +77,77 @@ static IncompleteKey fromPb(com.google.datastore.v1.Key keyPb) {
    * Returns the key's parent.
    */
   @Override
+  @Deprecated
   public Key parent() {
```

Comment:
The methods renamed are very simple (getters and setters in most cases) so I didn't see the need for calling the renamed one. I will updated the PRs accordingly though.